xml/System.Reflection.Metadata/TypeName.xml

```diff
@@ -458,17 +455,13 @@
       </ReturnValue>
       <Docs>
         <summary>
-          <para>Gets a value that indicates whether this is a "plain" type; that is, not an array, not a pointer, not a reference, and</para>
-          <para>not a constructed generic type. Examples of elemental types are "System.Int32",</para>
-          <para>"System.Uri", and "YourNamespace.YourClass".</para>
+          <para>Gets a value that indicates whether this is a "plain" type; that is, not an array, not a pointer, not a reference, and not a constructed generic type. Examples of elemental types are "System.Int32", "System.Uri", and "YourNamespace.YourClass".</para>
```

Comment:
When a `<summary>` or `<remarks>` or other containing element is reduced down to a single paragraph, the `<para>` tag is superfluous and the content can just be directly in the containing tag.